cluster/agent/stacks/09-amplifier.ampcore.yml

```diff
@@ -38,10 +38,10 @@ services:
         constraints:
         - node.labels.amp.type.api == true
     secrets:
-      - source: default.amplifier.yml
+      - source: amplifier_yml
```

Comment:
I'd actually rather fix the name in the places where we're creating the secret (travis, etc) since it makes the intention clearer, but I guess we can do that another time.